packages/extension-base/src/background/types.ts

```diff
@@ -306,8 +306,8 @@ export interface RequestSigningIsLocked {
 }
 
 export interface ResponseSigningIsLocked {
-  isLocked: boolean;
   remainingTime: number;
+  isLocked: boolean;
```

Comment:
same as above - I thought we were removing this

Reply:
answered on previous comment